python/tvm/auto_scheduler/measure.py

```diff
@@ -701,7 +700,7 @@ def local_builder_build(inputs, timeout, n_parallel, build_func="default", verbo
         [
             (
                 i.serialize(),
-                build_func,
+                BuildFunc.build_func,
```

Comment:
If we can passing the callable function here, should we also change the type of this argument from `str` to `callable` at line 655?

Reply:
Good catch, could you send a PR?